lib/Liquibase.js

```diff
@@ -17,43 +17,91 @@ class Liquibase {
    */
     constructor(settings) {
         this.settings = settings
-        this.liquibaseJarFile = '../liquibase-core-3.5.3.jar'
-        this.liquibaseDownloadSite='https://repo1.maven.org/maven2/org/liquibase/liquibase-core/3.5.3/liquibase-core-3.5.3.jar' 
+        this.liquibaseZipFile = '/tmp/liquibase/liquibase-3.8.0-bin.tar.gz'
+        this.liquibaseHomeDir = '/tmp/liquibase/'
+        this.liquibaseDownloadSite='https://github.com/liquibase/liquibase/releases/download/liquibase-parent-3.8.0/liquibase-3.8.0-bin.tar.gz'
+        this.jdbcDrivers = [{groupId:'com.oracle.jdbc', artifactId: 'ojdbc8', version: '18.3.0.0'}] ;
       }
-  
-    install(){
-      const credentials = this.settings.credentials
-      const mavenRepository = new MavenRepository('https://bwa.nrs.gov.bc.ca/int/artifactory/libs-release', credentials);
-      return mavenRepository.cache({groupId:'org.liquibase', artifactId:'liquibase-core', version:'3.5.3'}).then((liquibaseJarFile) =>{
-        const promises = []
-        if (this.settings.drivers){
-          for (let driver of this.settings.drivers){
-            promises.push(mavenRepository.cache(driver))
-          }
-        }
 
-        return Promise.all(promises).then((drivers) => {
-            return {liquibase: liquibaseJarFile, drivers:drivers};
+      downloadDrivers (){
+        const drivers = []
+        if (this.jdbcDrivers){
+           const mavenRepository = new MavenRepository('https://bwa.nrs.gov.bc.ca/int/artifactory/libs-release', this.settings.credentials);
+           for (let jdbcDriverGAV of this.jdbcDrivers){
+              drivers.push(mavenRepository.cache(jdbcDriverGAV));
+           }
+        }
+        const process = spawn('cp',[drivers[0], `${this.liquibaseHomeDir}/lib`]);
+        process.on("exit", function (code, signal) {
+          console.log("exit")
```

Comment:
Probably is useful to give meaningful statement, like "exited copy successfully"